src/components/icons.tsx

```diff
@@ -40,7 +40,7 @@ export const DownUpArrowsIcon = (props: IconProps) => {
   return <Icon as={HiArrowsUpDown} boxSize={6} {...props} />
 }
 export const KebabVertical = (props: IconProps) => {
-  return <Icon as={GoKebabVertical} w={3} h={6} {...props} />
+  return <Icon as={BsThreeDotsVertical} w={3} h={6} {...props} />
```

Comment:
What is the significance of this change?

Reply:
When updating a library, the GoKebabVertical didn't exist anymore. I checked out the page and couldn't find a reference to it in current versions, so I changed it to something that resembled the original icon the closest possible